book/src/dev/rfcs/0003-depdendent-verification.md

```diff
@@ -0,0 +1,373 @@
+# Dependent Verification and State Updates
```

Comment:
I think the filename of the RFC has a typo:
![image](https://user-images.githubusercontent.com/552961/91786076-d2c5e480-ebd4-11ea-9316-2a0d8381117b.png)